servers/quarkus-server/src/testFixtures/java/org/projectnessie/server/catalog/AbstractIcebergCatalogTests.java

```diff
@@ -204,6 +207,16 @@ public void testRegisterTableFromFileSystem() throws Exception {
     Assertions.assertThat(catalog.tableExists(TABLE)).isFalse();
   }
 
+  protected FileIO temporaryFileIO(RESTCatalog catalog) {
```

Comment:
Abstract to be reusable in `ITS3IcebergCatalog`?

Reply:
We would need to implement this method in: 

- TestAdlsIcebergCatalog (org.projectnessie.server.catalog)
- TestS3IcebergCatalog (org.projectnessie.server.catalog)
- TestGcsIcebergCatalog (org.projectnessie.server.catalog)
- ITGcsIcebergCatalog (org.projectnessie.server.catalog.gcs)
- ITS3IcebergCatalog (org.projectnessie.server.catalog.s3)
- ITOAuthIcebergCatalog (org.projectnessie.server.catalog.auth)
- ITBearerTokenIcebergCatalog (org.projectnessie.server.catalog.auth)

Not sure if the duplication is worth it.

Reply:
It is already overridden in `ITS3IcebergCatalog` - the only difference is the access-key-id and secret-access-key.
It's okay though as it is, if `ITS3IcebergCatalog` is the only overridden one.